pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -427,87 +457,93 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 	if err != nil {
 		r.status.SetDegraded(operatorv1.ResourceValidationError, fmt.Sprintf("Failed to retrieve / validate  %s", render.TigeraLinseedSecret), err, reqLogger)
 		return reconcile.Result{}, err
-	} else if esgwCertificate == nil {
+	} else if linseedCertificate == nil {
 		log.Info("Linseed certificate is not available yet, waiting until they become available")
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Linseed certificate are not available yet, waiting until they become available", nil, reqLogger)
 		return reconcile.Result{}, nil
 	}
 
+	trustedBundleCerts := []certificatemanagement.CertificateInterface{linseedCertificate}
+	if r.enterpriseCRDExists {
+		trustedBundleCerts = append(trustedBundleCerts, prometheusCertificate, esgwCertificate)
+	}
 	// Fluentd needs to mount system certificates in the case where Splunk, Syslog or AWS are used.
-	trustedBundle, err := certificateManager.CreateTrustedBundleWithSystemRootCertificates(prometheusCertificate, esgwCertificate, linseedCertificate)
+	trustedBundle, err := certificateManager.CreateTrustedBundleWithSystemRootCertificates(trustedBundleCerts...)
 	if err != nil {
 		r.status.SetDegraded(operatorv1.ResourceCreateError, "Unable to create tigera-ca-bundle configmap", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
 	certificateManager.AddToStatusManager(r.status, render.LogCollectorNamespace)
 
-	exportLogs := utils.IsFeatureActive(license, common.ExportLogsFeature)
-	if !exportLogs && instance.Spec.AdditionalStores != nil {
-		r.status.SetDegraded(operatorv1.ResourceValidationError, "Feature is not active - License does not support feature: export-logs", nil, reqLogger)
-		return reconcile.Result{}, err
-	}
-
 	var s3Credential *render.S3Credential
-	if instance.Spec.AdditionalStores != nil {
-		if instance.Spec.AdditionalStores.S3 != nil {
-			s3Credential, err = getS3Credential(r.client)
-			if err != nil {
-				r.status.SetDegraded(operatorv1.ResourceValidationError, "Error with S3 credential secret", err, reqLogger)
-				return reconcile.Result{}, err
-			}
-			if s3Credential == nil {
-				r.status.SetDegraded(operatorv1.ResourceNotFound, "S3 credential secret does not exist", nil, reqLogger)
-				return reconcile.Result{}, nil
-			}
+	var splunkCredential *render.SplunkCredential
+	var useSyslogCertificate bool
+	if r.enterpriseCRDExists {
```

Comment:
Ok, one last time but you get the idea. If there are any more enterpriseCRDExists in the reconcile they should probably be checking for the Variant from the Installation resource.